### DISCUSSION TITLE
BUG: System Prompt

### DISCUSSION THREAD

**Discussion by DirtyKnightForVi** [open-webui/open-webui#2522] [General]
In the new version, the prompt words set under Settings --- General --- System Prompt do not work. However, the prompt words set under "Workspace --- Playground" do work. My machine is completely offline; could there be a setting issue somewhere?

**Comment by tjbck** [comment]
Try pulling the latest, we've added couple of fixes since our 0.1.125 release!

**Reply by DirtyKnightForVi** [reply]
Version 3.11 do not work again，and also Playground.

**Reply by DirtyKnightForVi** [reply]
more imformation : discussions #4375
